### ISSUE TITLE
Fix the "forked from ..."

### ISSUE THREAD

**Issue by radare** [radare/radare2book#112]
![screen shot 2018-07-17 at 13 17 51](https://user-images.githubusercontent.com/917142/42814221-d1bf774c-89c3-11e8-9f27-fd22e4a281df.png)

**Comment by XVilka** [comment]
I don't think this is fixable without recreating book. And anyway it is not appearing in the printed version, so why do you care?

**Comment by ITAYC0HEN** [comment]
I think that pinging the Github folks will fix this. It's a known thing
https://stackoverflow.com/a/16052845/7193722

**Comment by Maijin** [comment]
I just asked support, we'll see

**Comment by Maijin** [comment]
We need to move that radareorg after too

**Comment by radare** [comment]
i can do that with a git push, but then the issues will not me moved.

agree on moving to radareorg/

**Comment by radare** [comment]
but moving the repo will break gitbook

**Comment by Maijin** [comment]
possible, but don't they support orgs by now?
